composer.json

```diff
@@ -40,7 +40,19 @@
     },
     "config": {
         "allow-plugins": {
-            "phpstan/extension-installer": true
+            "phpstan/extension-installer": true,
+            "symfony/flex": true
+        }
+    },
+    "extra": {
+        "symfony": {
+            "require": "7.0"
```

Comment:
This seems to be handled by `SYMFONY_REQUIRE` in CI? I think you can remove it too?